src/intl/en/page-wallets.json

```diff
@@ -53,8 +53,10 @@
   "page-wallets-try-dapps": "Try some dapps",
   "page-wallets-try-dapps-alt": "An illustration of Ethereum community members working together",
   "page-wallets-try-dapps-desc": "Dapps are applications built on Ethereum. They’re cheaper, fairer and kinder on your data than most traditional applications.",
-  "page-wallets-types": "Types of wallet",
-  "page-wallets-web-browser": "Web wallets that let you interact with your account via a web browser",
+  "page-wallets-types": "Types of wallet interfaces",
```

Comment:
Hmmm so what is a "wallet" vs. what is a "wallet interface"? This now confuses me a bit.

Reply:
To me these are still "types of wallets". Not convinced we should introduce yet more jargon.